mod.ts

```diff
@@ -1039,7 +1039,7 @@ const SDL_KeyboardEvent = new PackedStruct({
   keysym: SDL_KeySym,
 });
```

Comment:
Remove padding from all structs now that they are aligned and automatically padded...